graph/devicedefinition.resolvers.go

```diff
@@ -0,0 +1,21 @@
+package graph
+
+// This file will be automatically regenerated based on the schema, any resolver implementations
+// will be copied through when generating and any unknown code will be moved to the end.
+// Code generated by github.com/99designs/gqlgen version v0.17.39
+
+import (
+	"context"
+
+	"github.com/DIMO-Network/identity-api/graph/model"
+)
+
+// Devicedefinition is the resolver for the devicedefinition field.
+func (r *queryResolver) Devicedefinition(ctx context.Context, by model.DevicedefinitionBy) (*model.DeviceDefinition, error) {
```

Comment:
We seem split in the Go and the GraphQL on whether it's `devicedefinition` or `deviceDefinition`. I would vote for `deviceDefinition`.

Reply:
The definition was updated